cadc-vos/src/main/java/ca/nrc/cadc/vos/Transfer.java

```diff
@@ -91,76 +88,46 @@
      */
     public int version = VOS.VOSPACE_20;
 
-    private URI target;
+    private List<URI> targets = new ArrayList<URI>();
```

Comment:
This variable must be public.  The following pattern should be followed:

Mandatory fields should be private and have setters and getters.  There's only 1 now: direction.

All others (that are expected to be set by clients) should be public and not have getters and setters.

Reply:
As discussed - targets and protocols member variables of Transfer.java are now public, setters and getters removed for those variables. All other member variables of Transfer.java are staying as-is.